spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -18,7 +18,10 @@
 	<properties>
 		<google-cloud-core.version>1.2.3</google-cloud-core.version>
 		<google-cloud-pubsub.version>0.20.1-beta</google-cloud-pubsub.version>
+		<google-cloud-trace.version>0.20.1-alpha</google-cloud-trace.version>
 		<google-cloud-storage.version>1.2.1</google-cloud-storage.version>
+		<cloud-trace-java.version>0.4.0</cloud-trace-java.version>
+		<spring-cloud-sleuth.version>1.3.0.M1</spring-cloud-sleuth.version>
```

Comment:
spring boot dependencies doesn't have spring cloud dependencies.  Doc suggests using Spring Cloud Sleuth's BOM to include the dependency: https://cloud.spring.io/spring-cloud-sleuth/

Although, Sleuth is also part of the Spring Cloud release train. I don't think we can dependency import from spring cloud release train though. Because spring-cloud-gcp would ultimately be part of it.

Reply:
@saturnism is right. You could either reference the Spring Cloud release train BOM and then you get the Sleuth version out of the box, or you provide the Spring Cloud Sleuth's BOM but you have to provide info which version of the BOM is it.